packages/react/src/react-chat.ts

```diff
@@ -1,67 +1,49 @@
-import { StandardSchemaV1, Validator } from '@ai-sdk/provider-utils';
 import {
   AbstractChat,
+  AbstractChatInit,
   ChatState,
   ChatStatus,
-  ChatTransport,
   InferUIDataParts,
   UIDataPartSchemas,
   UIDataTypes,
   UIMessage,
-  UseChatOptions,
 } from 'ai';
 
 class ReactChatState<MESSAGE_METADATA, DATA_TYPES extends UIDataTypes>
   implements ChatState<MESSAGE_METADATA, DATA_TYPES>
 {
-  private messages: UIMessage<MESSAGE_METADATA, DATA_TYPES>[];
-  private status: ChatStatus = 'ready';
-  private error: Error | undefined = undefined;
+  #messages: UIMessage<MESSAGE_METADATA, DATA_TYPES>[];
+  status: ChatStatus = 'ready';
```

Comment:
because these already behaved like public properties, they can now just _be_ public properties. `messages` only can't because we have to make sure sets to it are immutable (by spreading the incoming array)